tile/TileRTL.py

```diff
@@ -123,12 +124,50 @@ def update_opt():
       s.crossbar.recv_opt.en   @= s.ctrl_mem.send_ctrl.en
       s.ctrl_mem.send_ctrl.rdy @= s.element.recv_opt.rdy & s.crossbar.recv_opt.rdy
 
-  # Line trace
-  def line_trace( s ):
+  # verbose trace if verbosity > 0

```

Comment:
Similar to the above `extract common part` review...

Reply:
done